### ISSUE TITLE
Python - Debugging  - JustMyCode Missing

### ISSUE THREAD

**Issue by wojtek-viirtue** [GoogleCloudPlatform/cloud-code-vscode#328]
**Use case**
Step through code that was installed as an external library.

**Feature**
The standard python debugging as provided by the python extension in VSCode allows for setting an option "justMyCode": false which enables stepping through code that isn't in a projects src mapping.

The current workaround I'm using is to port-forward the pod's debugging port locally and then using the following to config in launch.json

```    
{
      "name": "Python: Remote Attach",
      "type": "python",
      "request": "attach",
      "connect": { "host": "localhost", "port": 5678 },
      "pathMappings": [
        { "localRoot": "${workspaceFolder}/src", "remoteRoot": "/app" }
      ],
      "justMyCode": false
}
```

Here is a standard config for a debugging session which works well except stepping through code:
```
{
	"name": "K8S - GRPC",
	"type": "cloudcode.kubernetes",
	"request": "attach",
	"language": "Python",
	"debugPort": 5678,
	"podSelector": {
		"app": "mypod"
	},
	"localRoot": "${workspaceFolder:taxes}/src",
	"remoteRoot": "/app"
}
```

In all honesty, the holy grail would be able to map multi-root workspaces which contain a deployed version of a library to the location of the library that's deployed in sys.path, but maybe I'm missing something super simple.

**Comment by kschaab** [comment]
Great suggestion!

**Comment by kschaab** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-vscode-internal](https://github.com/GoogleCloudPlatform/cloud-code-vscode-internal/issues/3579)

**Comment by quoctruong** [comment]
@wojtek-viirtue For your debug scenarios, are you using `launch` mostly or `attach`?

**Comment by wojtek-viirtue** [comment]
@quoctruong we mostly use attach given that we want to remotely debug containers running on our local minikube instances.

**Comment by quoctruong** [comment]
Got it. What do you think about `justMyCode` field in the `attach` configuration for `cloudcode.kubernetes`? Does that work for you?

**Comment by wojtek-viirtue** [comment]
@quoctruong That'd be perfect!  Thank you.

**Comment by wojtek-viirtue** [comment]
@sivakku was this implemented in GoogleCloudPlatform/cloud-code-vscode-internal ?  Just wanted to follow up on the reason for the close.

**Comment by sivakku** [comment]
@wojtek-viirtue - Yes this is fixed in yesterday plugin release v1.8.0, do let me know your feedback after using it. Usually the one's tagged with "in-next-release" are closed right after the release.
